examples/basic/variables.tf

```diff
@@ -1,3 +1,9 @@
+variable "region" {
+  type    = string
+  default = "eu-west-1"
+}
```

Comment:
I'd rather not hardcode a default - should be a better choice for a random user whereever in the world.

Reply:
This is just in the example so the test will pass.

Reply:
I think it makes sense to have a default region, so that `terraform apply` doesn't ask about the region when you run it. One can specify `TF_VAR_region=us-east-1 terraform apply` to override if necessary.